### PR TITLE
Cart management

### PR DESCRIPTION
**Pull request by darshandamre**
added cart management

user can add products to cart, remove from cart, move to cart and move to wishlist

[Preview](https://52shop-web-git-cart-management-darshandamre.vercel.app)

### CODE REVIEW

File: src/pages/Cart/Cart.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
+.cart__content {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: flex-start;
+  justify-content: center;
```
Comment (nakulsharma15) [inline src/pages/Cart/Cart.css:5]: You can use a utility class here so that it can be used anywhere else as well.

File: src/pages/Cart/Cart.css [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+.cart__details__title {
+  padding-bottom: 1rem;
+  border-bottom: 1px solid var(--grey);
+}
+
+.price__item {
+  margin: 1rem 0;
+  display: flex;
+  justify-content: space-between;
```
Comment (nakulsharma15) [inline src/pages/Cart/Cart.css:23]: Try to use utility class here as this can be used elsewhere as well.

**Review by nakulsharma15 — APPROVED** [review]
Suggested some minor changes. Apart from that, Amazing work. I liked the folder structure and the way you named the files, variables and functions

**Review by tanishk15sharma — APPROVED** [review]
Naming conventions are good

**Pull request merged**